zoo/services/forms.py

```diff
@@ -87,6 +95,24 @@ class Meta:
             "tags": TagInput(),
         }
 
+    def __init__(self, *args, **kwargs):
+        if "instance" in kwargs and kwargs["instance"].repository:
+            kwargs["initial"] = {
+                "exclusions": kwargs["instance"].repository.exclusions or ""
+            }
```

Comment:
suggestion: I think this could be simplified like this, so it's a bit easier to read in my opinion. What do you think?

```suggestion
        instance = kwargs.get("instance")
        exclusions = instance.repository.exclusions if instance else ""

        kwargs.update({"initial": {"exclusions": exclusions}})
```

Reply:
oops... `repository` isn't a required field in service... so it will need an extra check